charts/wga/templates/daemonset.yaml

```diff
@@ -32,12 +32,6 @@ spec:
             limits:
               memory: "128Mi"
               cpu: "500m"
-            env:
-              - name: GOMEMLIMIT
```

Comment:
GOMEMLIMIT is a *soft* limit and indicates when the garbage collector has to start becoming more aggresive, so it's actually counterproductive to set it to the actual limit